bzt/modules/selenium.py

```diff
@@ -216,6 +216,8 @@ def _register_reader(self, report_file):
         return reader
 
     def prepare(self):
+        if self.execution.get(ScenarioExecutor.CONCURR, 1) > 1:
```

Comment:
You should use "get_load" instead, this is the common way to access load parameters. Otherwise you'll have to duplicate the logic inside it.